native/cocos/bindings/manual/jsb_spine_manual.cpp

```diff
@@ -233,6 +233,7 @@ bool register_all_spine_manual(se::Object *obj) {
     ns->defineFunction("disposeSkeletonData", _SE(js_register_spine_disposeSkeletonData));
 
     spine::setSpineObjectDisposeCallback([](void *spineObj) {
+#if CC_PLATFORM != CC_PLATFORM_OPENHARMONY
```

Comment:
? could not support spine?

Reply:
It should be supported, but there should still be problems

Reply:
Set USE_SPINE=OFF instead of modify these source files.